### DISCUSSION TITLE
Managing backpressure for publish

### DISCUSSION THREAD

**Discussion by Pakkes** [uNetworking/uWebSockets.js#292] [Q&A] [answered]
Hi.
In first, really really many thanks for this awesome library, which is simple, fast and reliable.
This is not an issue but a question, so please feel free to modify/tag this post properly (since I don't know how to tag this as a question).
I've understand the way we should manage the backpressure on ws.send ... but is there the same possibility when I use app.publish? May I expect the raise of backpressure also on app.publish?
If the answer is "yes", how can I take care about this?
Regards

**Comment by ghost** [comment]
You have maxBackpressure option which is the limit for when the subscribing connection is simply dropped

**Comment by Pakkes** [comment]
Many thanks for the immediate reply! I understand ... however, is there a way to take care about this by writing proper code to delay the publishing in case of backpressure? I would be tempted to increase this limit let's say up to 16 Kb per connection .... is this the proper way to (try to) manage backpressure on publish? or I will simply get dropped connections ?

**Comment by ghost** [comment, marked as answer]
I can only imagine two different strategies here.

Either you drop slow receivers, or you drop outgoing messages to slow receivers

You have to drop something
